maildirwatch.py

```diff
@@ -230,16 +290,18 @@ def _handle_file_event(self, _file_monitor, file, _other_file, event_type):
 
         try:
             with open(path) as inputfile:
-                message = email.message_from_file(
-                    inputfile, policy=email.policy.SMTP)
+                message = email.message_from_file(inputfile,
+                                                  policy=email.policy.SMTP)
                 self._queue.append(message)
         except FileNotFoundError:
             logger.error('Message file not found: %s', path)
             return
+        self._handle_messages()
 
-        if self._timer is not None:
-            GLib.source_remove(self._timer)
-        self._timer = GLib.timeout_add_seconds(60.0, self._handle_messages)
```

Comment:
Why do you use `Timer` from `threading` instead of GLib's timer?
`Timer` from threading is not thread safe AFAIK, and you're not using synchronization primitives.

Reply:
> Why do you use Timer from threading instead of GLib's timer?
It's because of interface segregation I guess. My point was that notifier should be responsible just for displaying notifications, not for synchronizing/handling duplicated messages. Moreover, this problem will arise in every notifier that will be developed in future, and IMHO it's not a notifier responsible. 

And of course I didn't want to use GLib timer in simple debug listener, because it's unnecessary dependency though. My objective is to develop extension for status bar email notifier based on maildirwatch, and it would be next notifier in list, and again - it shouldn't require GLib at all. I'll try to add synchronization to standard python Timer - is it ok for you? 

I'll try to generate yapf.cfg.
Thanks.

Reply:
> And of course I didn't want to use GLib timer in simple debug listener, because it's unnecessary dependency

But GLib is already a dependency - it's used for it's file notifications library, but if you properly synchronize `Timer` then it's fine with me. 

Nevertheless, I believe it's simpler to just use GLib's timers.

Reply:
Eventually I've used GLib notifier cause it's simpler.